sdk/cdn/Microsoft.Azure.Management.Cdn/src/Microsoft.Azure.Management.Cdn.csproj

```diff
@@ -5,7 +5,7 @@
 
 	<PropertyGroup>
 		<Description>Microsoft Azure CDN Management Library</Description>
-		<Version>6.1.0</Version>
+		<Version>6.2.0</Version>
```

Comment:
Hi @ChenglongLiu , why has the version been increased if only tests and recordings are being added? Thank you

Reply:
Thanks for the comment @nisha-bhatia, reverted the version.